src/System.Device.Gpio/System/Device/Gpio/Drivers/UnixDriver.Linux.cs

```diff
@@ -83,7 +83,7 @@ protected internal override void SetPinMode(int pinNumber, PinMode mode)
             }
             else
             {
-                throw new InvalidOperationException("There was an attempt to set a mode to a pin that is not yet open.");
+                throw new InvalidOperationException("There was an attempt to set a mode to a pin that is not open.");
```

Comment:
NOTE: Eventually I want all of these to be coming from a resources file instead to avoid duplication. I'll log an issue to track that work.